pkg/cvo/metrics.go

```diff
@@ -78,6 +82,10 @@ version for 'cluster', or empty for 'initial'.
 			Name: "cluster_operator_condition_transitions",
 			Help: "Reports the number of times that a condition on a cluster operator changes status",
 		}, []string{"name", "condition"}),
+		clusterInstaller: prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Name: "cluster_installer",
+			Help: "Reports the installation method for the cluster",
```

Comment:
Is this supposed to have UPI vs IPI?

Reply:
I think you'd want to return 1 / 0 if this is about IPI vs UPI, or 1 always if you have a label for UPI.  Recommend documenting your labels in this.

Reply:
Well, it's a bit more than that. You can derive UPI vs IPI from it, but you can also look at what invoked the installer (e.g. Hive). I can filled out that help text a bit more though.

Reply:
Wait, so now I'm confused.  Is type=UPI the label?  Expected to see a test with that as an example (just so we know).

Might want to update metrics.md too once you finalize.

Reply:
I've updated metrics.md. Can you let me know if it makes sense now?